issues/usePatientInfo.ts

```diff
@@ -40,13 +72,38 @@ export default function usePatientInfo(
     );
 
   useEffect(() => {
-    userService.getUserInfo(userInfo.userId).then(setUserInfo);
+    userService.getUserInfo(userInfo.userId).then(() => {
+      if (isMounted()) {
+        setUserInfo;
+      }
+    });
+  }, [userId]); //should update userINfo State then other userEffects will render
```

Comment:
The `useMountedState` hook is a bit overengineering.

You can manage the "mounted" state this easily:

```
useEffect(() => {
  let isMounted = true;
  
  // do something
  
  return () => {isMounted = false};
}, [...]);
```